spring-rabbit/src/test/java/org/springframework/amqp/rabbit/test/RabbitMatchers.java

```diff
@@ -39,7 +39,7 @@ private RabbitMatchers() {
 
 		private final String pattern;
 
-		private RegexMatcher(String pattern) {
+		RegexMatcher(String pattern) {
```

Comment:
Enclosing `RabbitMatchers ` class has `private` ctor.
Omission?
I can fix on merge.

Were you accurate with inner classes, but `public`? :wink: 
https://github.com/spring-projects/spring-integration/commit/9ae8f329d7a4aed25d9b207a1628878f4a0989f3